ionic.config.json

```diff
@@ -3,6 +3,5 @@
   "integrations": {
```

Comment:
We really need to stop changing this file. I suspect there is a version difference in the Ionic CLI on our machines which causes this issue. At the very least do not commit this file and simply discard it.